mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlClient.java

```diff
@@ -80,6 +94,27 @@ public static void createTargetTable(String targetTable) throws SQLException, Cl
     }
   }
 
+  public static void createSourceDatatypesTable(String sourceTable) throws SQLException, ClassNotFoundException {
+    try (Connection connect = getMysqlConnection(); Statement statement = connect.createStatement()) {
```

Comment:
Cosmetic: Post the ';' char use new line.

Reply:
Done